src/components/Breadcrumb/Breadcrumb.jsx

```diff
@@ -114,7 +121,7 @@ const Breadcrumb = ({ children, className, hasOverflow, ...other }) => {
       ref={browserSupportsResizeObserver ? breadcrumbRef : null}
       data-testid="overflow"
     >
-      {hasOverflow ? (
+      {hasOverflow && browserSupportsResizeObserver ? (
```

Comment:
There a several hasOverflow checks in the code, so, what I would do is right after (and outside)  the `if (__DEV__)` check I would just reassign the hasOverflow.

`hasOverflow = browserSupportsResizeObserver && hasOverflow;`

That way all the code checks for hasOverflow will be false when `browserSupportResizeObserver` is false.

Reply:
Implemented suggested change